test/notify.spec.js

```diff
@@ -45,33 +45,61 @@ describe('TEST ALL NOTIFICATION ENDPOINTS', () => {
   it('should get one notifications', (done) => {
     chai
       .request(app)
-      .get('/api/v1/notifications/8')
+      .get('/api/v1/notifications/2')
       .set('authorization', userToken)
       .end((err, res) => {
         expect(res.body.status).to.equal('success');
-        expect(res.body.data.user);
+        expect(res.body).to.have.property('notify');
```

Comment:
I think you should equally try to test the actual content of the `notify` property

Reply:
I have updated the test. It now includes the content of the notify property